### PR TITLE
Build bindings

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -27,15 +27,15 @@ tasks:
       maxRunTime: 14400
       image: "ubuntu:14.04"
       env:
-        TENSORFLOW_BUILD_ARTIFACT: https://index.taskcluster.net/v1/task/project.deepspeech.tensorflow.pip.v1.0.0-warpctc.gpu/artifacts/public/home.tar.xz
+        TENSORFLOW_BUILD_ARTIFACT: https://queue.taskcluster.net/v1/task/CUfR_gCURKy7IPnZzpqTxg/runs/0/artifacts/public/home.tar.xz
```
Comment (lissyx) [inline .taskcluster.yml]: That's temporary URL, will be changed with a stable one once merged

File: .taskcluster.yml [hunk 2]
```diff
@@ -27,15 +27,15 @@ tasks:
...
       command:
         - "/bin/bash"
         - "--login"
         - "-cxe"
         - >
           apt-get -qq update && apt-get -qq -y install git pixz wget pkg-config libsox-dev &&
           adduser --system --home /home/build-user build-user &&
-          cd /home/build-user/ &&
+          cd /home/ &&
```
Comment (lissyx) [inline .taskcluster.yml]: To simplify between OSX and Linux, the tar that was made previously from /home/build-user/ is now made from /home

**Comment by dpk1228** [comment]
can anyone let me know, how to test my wav file using deepspeech python package.
i want to use built in trained model.
thanks

**Comment by kdavis-mozilla** [comment]
@dpk1228 please refer to the README

**Pull request merged**